upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -592,7 +593,18 @@ func findAutoscalingGroupLaunchConfiguration(g *autoscaling.Group) (string, erro
 			if g.MixedInstancesPolicy.LaunchTemplate.LaunchTemplateSpecification != nil {
 				// honestly!!
 				name = aws.StringValue(g.MixedInstancesPolicy.LaunchTemplate.LaunchTemplateSpecification.LaunchTemplateName)
-				version := aws.StringValue(g.MixedInstancesPolicy.LaunchTemplate.LaunchTemplateSpecification.Version)
+				request := &ec2.DescribeLaunchTemplateVersionsInput{
+					LaunchTemplateName: &name,
+				}
+				versions, _ := c.EC2().DescribeLaunchTemplateVersions(request)
```

Comment:
ignoring error is not good thing, maybe we should even print that?

Reply:
Thanks for the reminder. I already add print error message. Could you review again, please?